upstairs/src/lib.rs

```diff
@@ -3763,6 +3796,7 @@ impl Upstairs {
         offset: Block,
         data: Bytes,
         sender: Option<std_mpsc::Sender<Result<(), CrucibleError>>>,
+        is_write_unwritten: bool,
```

Comment:
I'd update the comment for this fn to explain what this flag / operation is.

Reply:
done